web/src/components/clustering/ClusteringCard.vue

```diff
@@ -2,96 +2,111 @@
+          Pair View
+        </v-tab>
+
+        <div class="empty-space" />
+
+        <v-tab v-if="cluster && showClusterTimeline">
+          Time Chart
+        </v-tab>
+
+        <v-tab>
+          Heatmap
+        </v-tab>
+
+        <v-tab>
+          Cluster
         </v-tab>
-        <v-tab-item></v-tab-item>
+      </v-tabs>
 
-        <div class="empty-space"></div>
+      <v-tabs-items v-model="activeTab">
+        <v-tab-item>
+          <!-- TODO: place pairs view here -->
+        </v-tab-item>
 
-        <v-tab v-if="cluster && showClusterTimeline(cluster)" :key="2"
-          >Time Chart</v-tab
-        >
-        <v-tab-item v-if="cluster && showClusterTimeline(cluster)">
+        <v-tab-item v-if="cluster && showClusterTimeline">
           <TimeSeriesCard :cluster="cluster" />
         </v-tab-item>
 
-        <v-tab :key="4"> Heatmap </v-tab>
-        <v-tab-item> <HeatMap :cluster="cluster" /> </v-tab-item>
+        <v-tab-item>
+          <HeatMap :cluster="cluster" />
+        </v-tab-item>
 
-        <v-tab :key="5"> Cluster </v-tab>
-        <v-tab-item> <GraphTab :cluster="cluster" /> </v-tab-item>
-      </v-tabs>
+        <v-tab-item>
+          <GraphTab :cluster="cluster" />
+        </v-tab-item>
+      </v-tabs-items>
     </v-expansion-panel-content>
   </v-expansion-panel>
 </template>
 
 <script lang="ts">
-import { Component, Vue, Prop } from "vue-property-decorator";
-import { Cluster } from "@/util/clustering-algorithms/ClusterTypes";
 import {
-  getAverageClusterSimilarity,
-  getClusterElements,
-  getClusterElementsArray,
-} from "@/util/clustering-algorithms/ClusterFunctions";
-import { File } from "@/api/models";
+  defineComponent,
+  PropType,
+  ref,
+  computed,
+  toRef,
+} from "@vue/composition-api";
+import { useCluster } from "@/composables";
+import { Cluster } from "@/util/clustering-algorithms/ClusterTypes";
+import { getClusterElementsArray } from "@/util/clustering-algorithms/ClusterFunctions";
 import HeatMap from "./HeatMap.vue";
-import DataTab from "./DataTab.vue";
 import GraphTab from "./GraphTab.vue";
 import TimeSeriesCard from "./TimeSeriesCard.vue";
-import ClusteringFileTag from "@/components/clustering/ClusteringFileTag.vue";
 import FileTagList from "@/components/clustering/FileTagList.vue";
 
-@Component({
+export default defineComponent({
+  props: {
+    cluster: {
+      type: Set as PropType<Cluster>,
+      required: true,
+    },
+  },
+
+  setup(props) {
+    const activeTab = ref(1);
+    const { clusterFiles } = useCluster(toRef(props, "cluster"));
+
+    // If the timeline should be shown.
+    // Timeline will only be shown if every cluster element has a timestamp.
+    const showClusterTimeline = computed(() => {
+      return getClusterElementsArray(props.cluster).every(
+        (e) => e.extra?.timestamp
+      );
+    });
+
+    // String of ids to show in the pair view.
+    // TODO: it may be better to inline the pair-table in the future for better UX.
```

Comment:
Can you make an issue instead of this TODO? This will make it easier to track this.